src/Symfony/Component/Console/Tests/Helper/QuestionHelperTest.php

```diff
@@ -402,6 +402,25 @@ public function testChoiceOutputFormattingQuestionForUtf8Keys()
         $dialog->ask($this->createInputInterfaceMock(), $output, $question);
     }
 
+    /**
+     * @expectedException        \Symfony\Component\Console\Exception\RuntimeException
+     * @expectedExceptionMessage Aborted
+     */
+    public function testAskThrowsExceptionOnMissingInputWithValidator()
+    {
+        $dialog = new QuestionHelper();
+        $dialog->setInputStream($this->getInputStream(''));
```

Comment:
Be careful when merging, this method has been deprecated so we need to remove this line and change `$this->createInputInterfaceMock()` by `$this->createStreamableInputInterfaceMock($this->getInputStream('')` below. Same for the test added in `SymfonyQuestionHelperTest`.